src/build_helper/lib.rs

```diff
@@ -28,6 +30,14 @@ macro_rules! t {
     };
 }
 
+/// Reads an environment variable and adds it to dependencies.
+/// Supposed to be used for all variables except those set for build scripts by cargo
+/// https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts
```

Comment:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-env-changed says this.
Apparently, if any of the "build script input" env vars changes, then cargo will rerun the script automatically regardless of any other `rerun-if` directives?